### DISCUSSION TITLE
Faceted / Distorted artefacts: FBX import / export

### DISCUSSION THREAD

**Discussion by TJPovey** [assimp/assimp#4063] [Q&A]
Assimp version 5.0.1

I've recently started using assimp to import an fbx file, do some modifications to each vertex of the meshes, then export back out to either fbx/obj.

I've noticed that after exporting the mesh, it is distorted / very faceted. I wondered if I was doing something wrong with exporting the mesh, so removed all unnecessary code and just exported the imported scene using the below:

```C++
#include <assimp/Importer.hpp>
#include <assimp/Exporter.hpp>
#include <assimp/scene.h>
#include <assimp/postprocess.h>

Assimp::Importer importer;
const std::string& pFile = "path/to/file.fbx"
const aiScene* scene = importer.ReadFile(pFile, aiProcess_EmbedTextures);

Assimp::Exporter exporter;
exporter.Export(scene, "fbx", "path/to/export/location/file.fbx");
```
The original fbx file:
![image](https://user-images.githubusercontent.com/51401278/131681246-3978ce58-3bf9-4875-824d-03a9ebef7be6.png)

The exported fbx file from assimp:
![image](https://user-images.githubusercontent.com/51401278/131681334-4c166abe-340b-41e9-8b61-d67452c429c2.png)

Any help is appreciated.
Thanks

**Comment by kimkulling** [comment]
Hm, could I get the file to analyze this issue?

**Reply by TJPovey** [reply]
Thanks for taking the time to look into this @kimkulling, if it's relevant, the file was creating using Autodeesk AutoCAD (.dwg), opened in Autodesk Navisworks and exported to fbx. I then import this fbx using assimp. 

Units are in meters, not sure if this is causing the issue.
[Test_File_Real_Coords.zip](https://github.com/assimp/assimp/files/7114461/Test_File_Real_Coords.zip)

**Reply by TJPovey** [reply]
Hi @kimkulling, could this be due to the model being far from the origin? If so, would it be possible to avoid this issue without moving the model closer to 0,0?
